R/enframe.R

```diff
@@ -1,10 +1,10 @@
-#' Converting atomic vectors to data frames
+#' Converting atomic vectors to data frames, and vice versa
```

Comment:
I'm guessing that will involve more than changing this documentation 😬? Suspect this merits a bit of discussion ... shall I open an issue for it?